nixos/modules/services/networking/rpki-client.nix

```
@@ -0,0 +1,108 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.rpki-client;
+  tals = [
+    "${cfg.package}/etc/rpki/afrinic.tal" "${cfg.package}/etc/rpki/apnic.tal"
+    "${cfg.package}/etc/rpki/lacnic.tal" "${cfg.package}/etc/rpki/ripe.tal" ]
+    ++ optional cfg.accept-arin-rpa "${pkgs.rpki-arin-tal}/arin.tal"
+    ++ cfg.additional-tals;
+in {
+  options = {
+    services.rpki-client = {
+      enable = mkEnableOption (lib.mdDoc "Enable OpenBSD rpki-client");
+
+      package = mkOption {
+        description = (lib.mdDoc "OpenBSD rpki-client");
+        type = types.package;
+        default = pkgs.rpki-client;
+        defaultText = literalExpression "pkgs.rpki-client";
+      };
+
+      accept-arin-rpa = mkOption {
+        description = (lib.mdDoc "Accept the ARIN RPA and download the ARIN TAL");
+	type = types.bool;
```

```suggestion
        type = types.bool;
```
Please replace all tabs indentenations with spaces.